webcomponents/tool/pom.xml

```diff
@@ -145,5 +149,30 @@
             </plugin>
         </plugins>
     </build>
-
+    <profiles>
+        <profile>
+            <id>skip-frontend-tests-when-skipTests</id>
+            <activation>
+                <property>
+                    <name>skipTests</name>
+                    <value>true</value>
+                </property>
+            </activation>
+            <properties>
+                <skip.frontend.tests>true</skip.frontend.tests>
+            </properties>
+        </profile>
+        <profile>
+            <id>skip-frontend-tests-when-maven-test-skip</id>
+            <activation>
+                <property>
+                    <name>maven.test.skip</name>
+                    <value>true</value>
+                </property>
+            </activation>
+            <properties>
+                <skip.frontend.tests>true</skip.frontend.tests>
+            </properties>
+        </profile>
+    </profiles>
```

Comment:
@kunaljaykam I think the profiles here are superfluous here as you would just use the option on the command line anyways.

Also I like having separate flags for skipping backend and frontend tests.

Reply:
you are write we don't need this. we can just use `-Dskip.npm=true`. thanks. i am closing this pr.